### DISCUSSION TITLE
Invalid version: undefined

### DISCUSSION THREAD

**Discussion by Joblyn** [parcel-bundler/parcel#6699] [General] [answered]
Throws the above error while starting my app
Here's a shot of the error and my `devDependencies`
![Screenshot from 2021-08-09 23-14-12](https://user-images.githubusercontent.com/62160764/128781218-d980a063-8727-4df6-bfc0-b8378b3a66bc.png)
 
Any help is appreciated

**Comment by DeMoorJasper** [comment, marked as answer]
Parcel 1 isn't really actively developed anymore. It's probably a good idea to migrate to Parcel 2 https://v2.parceljs.org/getting-started/migration/

From your dependencies it however does seem like you're mixing babel 7 and babel 6 plugins which aren't compatible. Which is probably causing this error
